tutorials/webpack/plugin.md

```diff
@@ -0,0 +1,52 @@
+Webpack's `DefinePlugin()` function lets you replace a given token in the compiled code with another token.
+A common use case is using it to define environment variables when you cannot use an `.env` file directly.
+
+```javascript
+'use strict';
+
+require('dotenv').config();
+
+const fs = require('fs');
+const webpack = require('webpack');
+
+const compiler = webpack({
+  mode: 'development',
```

Comment:
Let's get rid of the unnecessary properties here. So no `mode`, `target, or `optimization`.